typescript/infra/config/environments/mainnet3/gasPrices.json

```diff
@@ -16,23 +16,23 @@
     "decimals": 9
```

Comment:
I didn't actually apply any changes onchain here or in tokenPrices yet, happy to undo these changes and just keep the SOON diff if that makes sense

Reply:
yeah would be good to keep just the SOON diff here - avoids confusion when looking back in history since i'll be updating this in https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/5116 too